uint8[] friendlyId
int8[] friendlyStatus
int8[3] friendlyPos
int8[] friendlyBatt
float64[] timestamp
